response_operations_ui/templates/home.html

```diff
@@ -1,16 +1,20 @@
 {% extends "layouts/base.html" %}
```

Comment:
Did these UI changes come through RAD? This home page is already in the RAD designs so this should probably be passed by @nicprice

Reply:
This page is identical except for the log out link (which will be moved to a more sensible place when the styling gets applied).

Reply:
I've updated the home page to look like the design.  Currently the scss is in the repo (and was given to me by leah) though there is talk of it using the cdn instead.  Short term though this isn't an issue.